README.md

```diff
@@ -51,3 +51,8 @@ Hamcrest matchers are easy to use as:
 ```php
 Hamcrest_MatcherAssert::assertThat('a', Hamcrest_Matchers::equalToIgnoringCase('A'));
 ```
+
+
+Documentation
+-------------
```

Comment:
The `-------------` looks like RST formatting, but this is Markdown file. Please change to Markdown format.